download-vector-tiles-to-local-cache/src/main/res/layout/activity_main.xml

```diff
@@ -34,24 +34,21 @@
 
             <com.google.android.material.button.MaterialButton
                 android:id="@+id/exportVectorTilesButton"
-                android:layout_width="match_parent"
+                style="@style/OutlineButton"
```

Comment:
could we also have the "Download Vector Tiles" button as a contained (regular) button and not outlined button?
Just to follow a standard set (Filled color button for primary action and outline button for secondary action) with other samples like Create planar and geodetic buffers, Apply function to raster from service, Clip geometry ..